charts/traefik-forward-auth/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik-forward-auth
 description: A minimal forward authentication service that provides OAuth/SSO login and authentication for the traefik reverse proxy/load balancer
 type: application
-version: 1.0.4
+version: 1.0.5
```

Comment:
This should not be required since it's only a documentation change.

Reply:
I was following https://github.com/k8s-at-home/charts/blob/master/CONTRIBUTING.md#immutability

> Chart releases must be immutable. Any change to a chart warrants a chart version bump even if it is only changed to the documentation.

I thought README.md was baked into the published chart artifacts.  WDYT?

Reply:
We've made some adjustments to the way the charts are released in that a version bump shouldn't need to be required for documentation changes. However, I don't think it's something we are looking to enforce though. @bjw-s what do you think?

Reply:
Agreed, not something we need to enforce, just wanted to share the info :)